File: Arrays/Python/First_and_last_occurrences_of_x.py
def find(arr,n,x):
    # code here
    a = []
    for i in range(n):
        if arr[i]==x:
            a.append(i)
    if len(a)==1:
        return a*2
    elif len(a)>1:
        return a[0],a[-1]
    else:
        return [-1,-1]

File: Arrays/Java/First_and_last_occurrences_of_X.java
class Solution{
    public ArrayList<Integer> firstAndLast(int arr[], int n, int x){
        // Code here
        ArrayList<Integer> list = new ArrayList<>();
        for(int i=0; i<n; i++){
            if (arr[i] == x) {
                list.add(i);
                break;
            }
        }
       
        for (int i=n-1; i>=0; i--) {
            if(arr[i] == x) {
                list.add(i);
                break;
            }
        }
       
        if (list.size() == 0) {
            list.add(-1);
            return list;
        }
        return list;
    }
}
